CONTRIBUTING.md

```diff
@@ -0,0 +1,117 @@
+# Contributing
+
+When contributing to Figura, please first discuss and/or interact with our community via our [Discord](https://discord.gg/figuramc). 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+note - do not merge upstream into your pr branch if a clean rebase is possible
+
+1. Ensure the feature is In demand via the `📧丨mod-suggestions` channel in our [Discord](https://discord.gg/figuramc) or has the [Enhancment]([https://github.com/FiguraMC/Figura/labels/enhancement](https://github.com/FiguraMC/Figura/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement+)) tag in the Issues tab 
+   
+    OR:
+
+   Ensure what you are doing is actually a bug via the `👾丨bug-reports` channel in our [Discord](https://discord.gg/figuramc) or has the [Bug]([https://github.com/FiguraMC/Figura/issues](https://github.com/FiguraMC/Figura/issues?q=is%3Aissue+is%3Aopen+label%3Abug+)) tag in the Issues tab 
+
+3. Communicate with Devs and other contributors about what you are building and adding. This helps keep multiple people from working on the same thing alone.
+
+4. Clearly and extensively document everything you've added/changed in the pull request. This helps when compiling all the changes into full release patch notes.
+
+5. You may merge the Pull Request in once you have the sign-off of another developer, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
```

Comment:
You may *not* merge Pull Request, without being a collaborator on the repository, "Reviewer" does not grant write access either. "Your pull request will be merged by one of the developers on their own discretion, constantly asking developers to accept your Pull Request *might* work, but is more likely to just annoy them and postpone the merge."